spec/plainyearmonth.html

```diff
@@ -249,10 +260,13 @@ <h1>Temporal.PlainYearMonth.prototype.add ( _temporalDurationLike_ [ , _options_
         1. Let _fieldNames_ be ? CalendarFields(_calendar_, « *"month"*, *"year"* »).
         1. Let _sign_ be ! DurationSign(_duration_.[[Years]], _duration_.[[Months]], _duration_.[[Weeks]], _balanceResult_.[[Days]], 0, 0, 0, 0, 0, 0).
         1. If _sign_ &lt; 0, then
-          1. Let _day_ be <mark>TODO: get via calendar.daysInMonth</mark> ! DaysInMonth(_yearMonth_.[[ISOYear]], _yearMonth_.[[ISOMonth]]).
+          1. Let _daysInMonth_ be ? Get(_calendar_, *"daysInMonth"*).
+          1. Let _day_ be ? Call(_daysInMonth_, _calendar_, « _yearMonth »).
```

Comment:
todo: ToInteger?

Reply:
I will address this in a follow up, since it seems like actually all these methods should be called by abstract operations that then call ToInteger on the result.